drivers/usbc/tcpc/ucpd_stm32gx.c

```diff
@@ -0,0 +1,1442 @@
+/*
```

Comment:
nit: filename uses `g`, but seems to apply to `L5/U5`

Reply:
I renamed the file stm32_ucpd.c